src/stores/useFormatter.ts

```
@@ -261,6 +261,10 @@ export const useFormatter = defineStore('formatter', {
           }
         }
 
+        if (denom && denom.startsWith('factory/')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)